src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -133,11 +134,12 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
    * @return a list of [[docs.DepositInfo]] objects
    */
   def getDeposits(user: String): Try[Seq[DepositInfo]] = {
+    implicit val timestampOrdering: Ordering[DateTime] = Ordering.fromLessThan[DateTime](_ isBefore _).reverse
     for {
       deposits <- DepositDir.list(draftBase, user)
       infos <- deposits.map(_.getDepositInfo(submitBase, easyHome)).collectResults
-    }
-      yield infos
+      sortedInfos = infos.sortBy(deposit => (deposit.state, deposit.date))
```

Comment:
Please move sorting into `DepositDir.list` and create unit tests.

Reply:
I can't! `DepositDir.list` only returns `Try[Seq[DepositDir]]`. Only in `DepositDir.getDepositInfo` you get the state from the `StateManager`. It is only after I got the state (in a `DepositInfo` object) that I can sort on `state.label` and `creation.timestamp`.

If you got a better suggestion of where to put this, please let me know!

Reply:
Thanks for setting up the unit test for this! Really helped with implementing the test itself.